spec/api/chemotion/reaction_api_spec.rb

```diff
@@ -0,0 +1,551 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Chemotion::ReactionAPI do
+  include_context 'api request authorization context'
+
+    let(:new_root_container) { create(:root_container) }
+
+  describe 'GET /api/v1/reactions' do
+    let!(:c1) do
+      create(:collection, label: 'C1', user: user, is_shared: false, reaction_detail_level: 10)
+    end
+    let!(:r1) { create(:reaction, name: 'r1', collections: [c1]) }
+    let!(:r2) { create(:reaction, name: 'r2', collections: [c1]) }
```

Comment:
Would it make sense to create a reaction in another collection so that it is checked that this is not delivered with it? Or are these already set with a seeding?

Reply:
See 515aebd.